### DISCUSSION TITLE
Is this project still alive?

### DISCUSSION THREAD

**Discussion by ductranit** [isar/isar#1581] [Help]
Hi there,
Firstly, I would like to express my appreciation for your excellent database library. It has proven to be a valuable resource.
I'm reaching out to inquire about the current status of the project, as there haven't been any updates in recent months. Is the project still active and being maintained?

Additionally, I'm particularly interested in the upcoming 4.0.0 release, specifically its features such as string ID and dynamic property handling. However, I understand that this version may not yet be ready for production use. 
Could you kindly provide an update or timeline for when we might expect this version to be production-ready?

Thanks!

**Comment by ndelanou** [comment]
Indeed, a status update would be greatly appreciated, even if it is bad news. 

Isar v3 is already amazing, and I am pleased with it. The promises of Isar v4 are extremely exciting, but I have also observed that the project has been left unmaintained recently. 

There is a pressing need for a prompt update on the project's maintenance to align everyone's expectations. 

I hope everyone is doing fine on the maintainer side

**Comment by vicenterusso** [comment]
You should try https://github.com/isar-community/isar/, unnoficial fork with updated dependencies and eventually bug fixes and new features

**Comment by pamafe1976** [comment]
Hi. 
Any news on the status of this project?

**Reply by vicenterusso** [reply]
No, stick with community fork and you are good

**Reply by abdullah-cse** [reply]
> No, stick with community fork and you are good

Does Community V4 support latest Flutter release?

**Comment by itxtoledo** [comment]
i think no, the github of main developer is inactive

**Comment by rmcsharry** [comment]
Simon Choi (main developer of Hive and Isar) has not posted on Twitter since Aug 2023 and not committed in github since January (6 months, which is very unlike him).

What is more worrying is that his LinkedIn page has been deleted.

@Jtplouffe do you know if Simon is ok?

**Reply by JakobLichterfeld** [reply]
> and not committed in github since January (6 months, which is very unlike him).

That's technically not correct, he committed in May 2024, even to isar: https://github.com/simc?tab=overview&from=2024-05-01&to=2024-05-31
